src/main/java/wooteco/subway/service/line/dto/LineResponse.java

```diff
@@ -39,6 +30,17 @@ public LineResponse(Long id, String name, LocalTime startTime, LocalTime endTime
         this.updatedAt = updatedAt;
     }
 
+    public static LineResponse of(Line line) {
+        return new LineResponse(line.getId(), line.getName(), line.getStartTime(),
+            line.getEndTime(), line.getIntervalTime(), line.getCreatedAt(), line.getUpdatedAt());
+    }
+
+    public static List<LineResponse> listOf(List<Line> lines) {
+        return lines.stream()
+            .map(it -> LineResponse.of(it))
```

Comment:
```suggestion
            .map(LineResponse::of)
```

메서드 레퍼런스를 사용하면 가독성이 상승합니다